pkg/neorpc/filters.go

```diff
@@ -112,7 +130,9 @@ func (f TxFilter) IsValid() error {
 	return nil
 }
 
-// Copy creates a deep copy of the NotificationFilter. It handles nil NotificationFilter correctly.
+// Copy creates a deep copy of the NotificationFilter.
+// If [NotificationFilter.ParametersSI] has been called before, cached values
```

Comment:
i can only move "If" to the previous line, otherwise it will be longer than 80 symbols. done and adjusted a little, but IMO 80 symbols wrapping is a good standard rule (it was like it before fixing)